CHANGELOG

```diff
@@ -1,6 +1,9 @@
 v3.10.3 (XXXX-XX-XX)
 --------------------
 
+* Activate RDB_CoveringIterator and use it for some geo index queries.
+  This speeds up and simplifies geo queries which do not use GEO_DISTANCE.
```

Comment:
NIT

"simplifies geo queries" with geo index

and "don't use GEO_DISTANCE or GEO_IN_RANGE"?

Reply:
As far as I see the geo index does not support `GEO_IN_RANGE` at all. I will add "with geo index".

Reply:
Ok, thanks!